components/phpunit_bridge.rst

```diff
@@ -798,6 +799,13 @@ classes, interfaces and/or traits for the code to run::
         }
     }
 
+Note that mocking a class with ``ClassExistsMock::withMockedClasses()``
+will make :phpfunction:`class_exists`, :phpfunction:`interface_exists`
+and :phpfunction:`trait_exists` return true.
+
+To register an enumeration and mock :phpfunction:`enum_exists`,
+``ClassExistsMock::withMockedEnums()`` must be used.
```

Comment:
perhaps it's obvious for ppl, but only yesteday i realized class_exists returns true for enums (https://github.com/symfony/symfony/pull/48516#issuecomment-1353261478)

so what this new method enables doing is:

```php
withMockedClasses([SomeClass::class => true])
withMockedEnums([SomeClass::class => false])
```

to trigger a code path for `if (class_exists(SomeClass::class) && !enum_exists(SomeClass::class)) {`